source/math/FloatBox2D.ooc

```diff
@@ -58,14 +58,14 @@ FloatBox2D: cover {
 	}
 	pad: func ~fromFloat (pad: Float) -> This { this pad(pad, pad, pad, pad) }
 	pad: func ~fromSize (pad: FloatSize2D) -> This { this pad(pad width, pad width, pad height, pad height) }
-	pad: func ~fraction (pad: Float) -> This {
-		this pad(pad * this size / 2.0f)
+	enlargeEvenly: func (fraction: Float) -> This {
```

Comment:
My only problem here is the name `enlargeEvenly`. It's not terrible, but it's not good either. A fix can be left for later, but then it will need a corresponding PR in depending projects, again.